spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/support/SimpleTaskRepository.java

```diff
@@ -36,6 +36,7 @@
 
 	public static final int MAX_EXIT_MESSAGE_SIZE = 2500;
 	public static final int MAX_TASK_NAME_SIZE = 100;
+	public static final int MAX_ERROR_MESSAGE_SIZE = 2500;
```

Comment:
My recommendation here is for the user to create their own TaskRepository.  Else they will have to set these via properties. And that didn't seem that clean to me.  So argue with me if you disagree.